boards/arm/efr32_slwstk6061a/doc/efr32_slwstk6061a.rst

```diff
@@ -145,6 +145,16 @@ Connect the EFR32-SLWSTK6061A to your host computer using the USB port and you
 should see a USB Serial Port. Use `J-Link`_ or Silicon Labs Simplicity Studio
 to flash the generated zephyr.bin.
 
+OpenOCD included in the Zephyr SDK v0.9.3 is too old and does not support the
+EFR32FG1P chip. You will need to compile and install the newest version of the
```

Comment:
Could you specify, which release version of OpenOCD is required?

Reply:
The OpenOCD is - unfortunately - released very infrequently and the last official release 0.10.0 is even older than the one we have in Zephyr SDK. I could mention specific commit which added support for EFR32FG1P chip but I believe that asking for 'newest version' is more straightforward. Maybe I should state it more explicitly that I'm talking about compiling the latest version of the source code and not installing the latest OpenOCD release?

I've already opened the issue to add latest OpenOCD to Zephyr SDK [#100](https://github.com/zephyrproject-rtos/meta-zephyr-sdk/issues/100) so hopefully this is only temporary.

Reply:
Yes, i think it would be good to clarify, that you are not meaning a release version, but some intermediate version from the source control system.

Reply:
I've updated the description.